### PR TITLE
feat: old design to carbon; user-interface to components

### PR DESCRIPTION
**Pull request by mbritense**
Closes https://ritense.tpondemand.com/entity/65702-fe-migratie-old-new-design-form and https://ritense.tpondemand.com/entity/65216-fe-merge-valtimouser-interface-to-valtimocomponents

### CODE REVIEW

File: projects/valtimo/components/src/lib/components/input-label/input-label.component.ts [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+
+  private readonly _title$ = new BehaviorSubject<string>('');
+  private readonly _titleTranslationKey$ = new BehaviorSubject<string>('');
+  private readonly _required$ = new BehaviorSubject<boolean>(false);
+
+  public readonly displayTitle$: Observable<string> = combineLatest([
+    this._title$,
+    this._titleTranslationKey$,
+    this._required$,
+    this.translateService.stream('key'),
```
Comment (teodor-ritense) [inline projects/valtimo/components/src/lib/components/input-label/input-label.component.ts:51]: I don't see that you're using this observable, please remove

File: projects/valtimo/components/src/lib/components/select/select.component.ts [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+
+  @Output() public selectedChange: EventEmitter<SelectedValue> = new EventEmitter();
+
+  public readonly selected$ = new BehaviorSubject<SelectedValue>('');
+  private readonly _listItems$ = new BehaviorSubject<Array<ListItem>>([]);
+
+  public readonly listItems$: Observable<Array<ListItem>> = combineLatest([
+    this._listItems$,
+    this.selected$,
+    this.translateService.stream('key'),
```
Comment (teodor-ritense) [inline projects/valtimo/components/src/lib/components/select/select.component.ts:76]: Same as before

File: projects/valtimo/components/src/lib/components/select/select.component.ts [hunk 3]
```diff
@@ -0,0 +1,180 @@
...
+          : selected === listItem.id;
+
+        return {
+          ...listItem,
+          selected: isSelected,
+          content: translation !== listItem.content ? translation : listItem.content,
+        };
+      })
+    ),
+    tap(items => console.log(items))
```
Comment (teodor-ritense) [inline projects/valtimo/components/src/lib/components/select/select.component.ts]: Remove console log

File: projects/valtimo/components/src/lib/components/select/select.component.ts [hunk 4]
```diff
@@ -0,0 +1,180 @@
...
+    ) {
+      this.setDefaultSelection();
+    }
+  }
+
+  ngOnDestroy() {
+    this._selectedSubscription?.unsubscribe();
+    this._clearSubjectSubscription?.unsubscribe();
+  }
+
```
Comment (teodor-ritense) [inline projects/valtimo/components/src/lib/components/select/select.component.ts]: Please bring in accordance to the coding guidelines

**Review by teodor-ritense — CHANGES_REQUESTED** [review]

File: projects/valtimo/components/src/lib/components/input-label/input-label.component.ts (hunk 1 above)
Comment (mbritense) [reply projects/valtimo/components/src/lib/components/input-label/input-label.component.ts:51]: am too

File: projects/valtimo/components/src/lib/components/select/select.component.ts (hunk 2 above)
Comment (mbritense) [reply projects/valtimo/components/src/lib/components/select/select.component.ts:76]: am too

File: projects/valtimo/components/src/lib/components/select/select.component.ts (hunk 3 above)
Comment (mbritense) [reply projects/valtimo/components/src/lib/components/select/select.component.ts]: done son

File: projects/valtimo/components/src/lib/components/select/select.component.ts (hunk 4 above)
Comment (mbritense) [reply projects/valtimo/components/src/lib/components/select/select.component.ts]: done son

**Review by teodor-ritense — APPROVED** [review]

**Pull request merged**